msal-dotnet-articles/advanced/webview2.md

```diff
@@ -5,44 +5,47 @@ description: "How to use the modern embedded browser based on Microsoft Edge wit
 
 # Using WebView2 with MSAL.NET
 
-A modern embedded browser based on Microsoft Edge, capable of performing Windows Hello, log-in with FIDO keys, etc. This browser replaces the old embedded WebView, based on an outdated version of Internet Explorer.
+WebView2 is a modern embedded browser runtime based on Microsoft Edge, capable of performing Windows Hello authentication, log in with FIDO keys, and more. This browser replaces the legacy web view based on Internet Explorer.
 
-## Where is it available?
+## Availability
 
-- All Windows versions
-- MSAL.NET version 4.28.0 and higher
-- [WebView2 runtime](/microsoft-edge/webview2/) must be installed on the machine.
+To use WebView2 in your application, the following requirements must be met:
 
-## Evergreen runtime
+- Windows 10 OS or newer (supported by the WebView2 runtime).
+- MSAL for .NET version 4.28.0 and higher.
+- [WebView2 runtime](/microsoft-edge/webview2/) must be installed on the machine.
 
-WebView2 runtime is available on most Windows 10 and Windows 11 machines by default. But it may not be available on older platforms.
+## Call pattern
 
-## Changes to call pattern
+When attempting to use the new WebView2 runtime, keep in mind the following required changes:
 
-- On .NET5-windows10.xxx, there is no change.
-- On .NET Classic and .NET Core 3.1, add a reference to [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/) and call `.WithDesktopFeatures()`.
+- In applications written against `.net5-windows10` (or newer), no changes are necessary.
+- In applications written against .NET Framework and .NET Core 3.1, add a reference to [`Microsoft.Identity.Client.Desktop`](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/) and add [`.WithWindowsEmbeddedBrowserSupport()`](xref:Microsoft.Identity.Client.Desktop.DesktopExtensions.WithWindowsEmbeddedBrowserSupport*) when instantiating the public client application.
 
 ```csharp
 var pca = PublicClientApplicationBuilder
-    .Create("client_id")
-    .WithDesktopFeatures()
-    .Build()
+        .Create("0f887a65-3c78-4b16-9529-6209b8741c26")
+        .WithWindowsEmbeddedBrowserSupport()
+        .Build();
 ```
 
+>[!IMPORTANT]
+>WebView 2 is **not supported** for Microsoft Entra ID (formerly known as Azure Active Directory) authorities. Using [`.WithWindowsEmbeddedBrowserSupport()`](xref:Microsoft.Identity.Client.Desktop.DesktopExtensions.WithWindowsEmbeddedBrowserSupport*) in that context is always going to default to the legacy web view, regardless of the setting provided during the instantiation of the public client application. This is caused by stability bugs identified during MSAL development. For B2C and Active Directory Federation Services (ADFS) authorities, WebView2 will be shown.
```

Comment:
This should probably be moved in the Behavior section under the table, as an asterisk.